doc/related_projects.rst

```diff
@@ -184,6 +184,12 @@ and tasks.
   K-means and mixture of von Mises Fisher clustering routines for data on the
   unit hypersphere.
 
+**Dataset resampling**
```

Comment:
True that this is only for classification.
I think that if a **Pre-processing** section would be there, it could go there.
Along, [`categorical-encoding`](https://github.com/scikit-learn-contrib/categorical-encoding) could go there.

WDYT?